routes/index.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express')
+const Router = express.Router()
+const db = require("../lib/database")
+
+module.exports = Router
+
+Router.get('/v1/questions', (req, res) => {
+  db.getAllQuestions()
+    .then((data) => {
+      res.json({data:data})
+    })
+    .catch((error) => console.log(error))
```

Comment:
Make sure you always send something back to the client! This will hang. Try `res.sendStatus(500)` as a minimum :)